1904-11-03.xml

```diff
@@ -0,0 +1,2652 @@
+                    <head>THE WAR.</head>
+                    <p>The only war news we have to-day is contained in a telegram from Tokio
+                        reporting a tremendous bombardment of Port Arthur, the fall of which is said
+                        to be near at hand. Meantime, all is quiet in Manchuria, though we may hear
+                        at any time that the Japanese have resumed the offensive. The annals of
+                        scientific warfare would be vainly searched for any tactical position re-
+                        sembling that of the two vast armies now facing one another between
+                        Liao-yang and Mukden. For fully two weeks, each of these immense forces has
+                        virtually stood at bay, challenging the other to come on. Pretty equally
+                        matched in numbers and pugnacity, they only wait for an order to rush at the
+                        foe, and obedience will be rendered with all possible alacrity. But Marshal
+                        Oyama and General Kuropatkin are concerned with matters of higher moment
+                        from their respective standpoints. It was the Russian late effort to march
+                        south, to relieve Port Arthur, which influenced the Japanese commander to
+                        replace offensive strategy by defensive. Similarly, it was the Japanese
+                        impetuous northerly advance that constrained Kuropatkin, after the failure
+                        of his attempt to get south, to halt on the Sha-ho. In the one case, the
+                        object is the capture of Port Arthur by starvation and storm combined before
+                        relief arrives; in the other, the Russians feel them- selves bound to retain
+                        possession of Mukden. at any cost. In effect, it is a game of patience, and
+                        it only remains to be seen which side will be the first to give way to
+                        irritation at the delay </p>
+                </div>
+            </div>
+
+            <pb n="3"/>
+            <!-- missing 3 items -->
+            <div type="item">
+                <head>THE KHEDIVE </head>
+                <p>The Khedive inspected his estates yesterday at Mariout, and returned in the
+                    evening to .Montazah palace, where he is spending today. The Gonvernorat of
+                    Cairo has issued invitations to numerous officials and notables to present
+                    themselves at the station at 11.30 a.m. next Tuesday to greet hisHighness on his
+                    arrival. The Ministry of War has given orders for a detachment to be' present as
+                    a guard of () The train in which his Highness travels will stop at Damanhour,
+                    Tantah, (Buket el) San, and Benha. </p>
+            </div>
+            <div type="advert">
```

Comment:
these three ads might also be templated? they'd have an xml:id attribute then

Reply:
They do not have an xml:id attribute because they are far off from what the templates were. It wouldn't match the template corresponding to the xml:id, thats why I did not put it in.